src/applications/personalization/hooks/useFeatureToggle.unit.spec.jsx

```diff
@@ -59,16 +50,19 @@ describe('useFeatureToggle hook', () => {
   });
 
   it('should throw an error if the toggle name is not found', () => {
-    const wrapper = renderInReduxProvider(
-      <TestComponent customToggleName="fakeToggleName" />,
-      {
-        initialState: {
-          featureToggles: {
-            profile_use_info_card: true,
+    try {
+      renderInReduxProvider(
+        <TestComponent customToggleName="fakeToggleName" />,
+        {
+          initialState: {
+            featureToggles: {
+              profile_use_info_card: true,
+            },
           },
         },
-      },
-    );
-    expect(wrapper.findAllByText('Toggle error:')).to.exist;
+      );
+    } catch (error) {
+      expect(error).to.exist;
+    }
```

Comment:
Some updates to testing the Toggler component and that it throws an error if you attempt to use a toggle that doesn't exist. Basically just testing cleanup, and increasing test coverage for error boundary.